### PR TITLE
fix: Reenabled patternfly quickstarts.

### PR DESCRIPTION
**Pull request by dlabaj**
Fixes issue #4251 by reenabling quickstarts.

### CODE REVIEW

**Review by evwilkin — CHANGES_REQUESTED** [review]
The `@patternfly/documentation-framework` dependency on [line 20](https://github.com/patternfly/patternfly-org/pull/4267/files#diff-877deda05a5e568e9dcfc173b0a9df126e74ffc0b86b2cf4cb3d7fa550d38e8cR20) should be bumped to `6.0.0-alpha.102` to match the [current docs-framework version](https://github.com/patternfly/patternfly-org/blob/main/packages/documentation-framework/package.json#L4) - this update resolved the CI build error for me locally.

File: yarn.lock [hunk 1]
```diff
@@ -1787,6 +1787,74 @@
     "@patternfly/react-core" "6.0.0-alpha.94"
     commander "^12.0.0"
 
+"@patternfly/documentation-framework@6.0.0-alpha.94":
```
Comment (nicolethoen) [inline yarn.lock:1790]: I think these lines in the yarn.lock file should be unnecessary. are they being pulled in if you run `yarn install` again?
Quickstarts shouldn't be requiring you to pull it in, I don't think.

**Comment by kmcfaul** [comment]
I think if you run `yarn install` and commit the `yarn.lock` that should clean it up.

**Review by evwilkin — APPROVED** [review]
LGTM! 🥇

File: yarn.lock (hunk 1 above)
Comment (dlabaj) [reply yarn.lock]: Good catch.  Thanks @nicolethoen

File: packages/documentation-site/package.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
     "@patternfly/react-user-feedback": "6.0.0-prerelease.1",
     "@patternfly/react-component-groups": "6.0.0-prerelease.3",
     "@patternfly/react-virtualized-extension": "6.0.0-prerelease.3",
+    "@patternfly/quickstarts": "6.0.0-alpha.5",
```
Comment (nicolethoen) [inline packages/documentation-site/package.json:29]: Did you want to pull in version `6.0.0-prerelease.1` of quickstarts? That'll include the updated design guidelines that Erin just finished.

**Review by nicolethoen — CHANGES_REQUESTED** [review]

**Review by kmcfaul — APPROVED** [review]

**Pull request merged**